source/docs/casper/developers/dapps/sdk/client-library-usage.md

```diff
@@ -271,3 +273,95 @@ print(deploy.hash.hex())
 </Tabs>
 
 Once submitted, the above snippet will print the deploy hash in the console.
+
+## Staking
+
+Token staking is vital to the Casper Network, enabling its delegated Proof-of-Stake consensus mechanism to operate effectively. With the help of any of the Casper SDKs, you can stake tokens by delegating them to validators.
+
+The delegation functionality exists as a piece of session code obtainable from the [casper-node](https://github.com/casper-network/casper-node) repository. To delegate tokens, first clone the repository:
+
+```bash
+git clone https://github.com/casper-network/casper-node.git
+cd casper-node/
+```
+
+Then compile the [delegate contract](https://github.com/casper-network/casper-node/blob/dev/smart_contracts/contracts/client/delegate/src/main.rs):
+
+```bash
+make setup-rs
+make build-contract-rs/delegate
+```
+
+Now, assuming that you cloned `casper-node` from your project's root directory, `cd` back into it:
+
+```bash
+cd ../
+```
+
+Now in your dApp's backend (or standalone script), load the *delegate.wasm* file into memory and deploy it with the arguments "amount", "delegator", and "validator" included.
+
+<Tabs>
+
+<TabItem value="js" label="JavaScript">
+
+```javascript
+const { CasperClient, Contracts, RuntimeArgs, CLValueBuilder, CLPublicKey } = require("casper-js-sdk")
+const fs = require("fs")
+
+const casperClient = new CasperClient("http://NODE_ADDRESS:7777/rpc")
+const contract = new Contracts.Contract(casperClient)
```

Comment:
```suggestion
const casperClient = new CasperClient("http://NODE_ADDRESS:7777");
const contract = new Contracts.Contract(casperClient);
```

Reply:
As far as I'm aware "/rpc" is required. Have you tested this and found otherwise?